Source/Dafny/AST/DafnyAst.cs

```diff
@@ -4000,52 +3996,53 @@ public class ModuleDefinition : INamedRegion, IAttributeBearingDeclaration {
     }
 
     VisibilityScope visibilityScope;
+    public VisibilityScope VisibilityScope =>
```

Comment:
With .NET source generators we should be able to write this as just 
```csharp
[Memoize] 
public VisibilityScope VisibilityScope => new VisibilityScope(this.SanitizedName)
```

But strangely it doesn't seem like there are NuGet packages with attributes like Memoize.

Reply:
Agreed, and I was disappointed to not find one :'(